gradle/versions.gradle

```diff
@@ -29,4 +29,11 @@ ext {
 
     ktlint_version = '0.36.0'
     android_sdk_versions_plugin = "1.0.0"
+
+    gms_google_services_version = '4.3.3'
+    firebase_crashlytics_plugin_version = '2.0.0-beta02'
```

Comment:
Are there stable versions though?

Reply:
I've seen just beta. But aren't we fearless?

Reply:
We are!